enterprise/reporting/event_stats.go

```diff
@@ -0,0 +1,49 @@
+package reporting
+
+import (
+	"strconv"
+
+	"github.com/rudderlabs/rudder-go-kit/stats"
+	"github.com/rudderlabs/rudder-server/utils/types"
+)
+
+const (
+	EventsDeliveredMetricName = "events_delivered_total"
+	EventsAbortedMetricName   = "events_aborted_total"
```

Comment:
Suggestion, would it be better if we use a single metric instead?

```suggestion
	EventsProcessedMetricName = "events_processed_total"
```
And have `status` as label/tag.

It would make it more convenient in the future if we want to capture other categories, plus the summary (delivered+aborted) is a useful number and can be used on it own or to create percentages